app/db_functions/personal.py

```diff
@@ -74,11 +74,66 @@ async def is_exist_item_db(text: str, context: Context.id) -> bool:
     return await Item.exists().where((Item.text == text) & (Item.context == context))
 
 
+async def is_words_in_card_db(telegram_user_id: int, item_relation_id: ItemRelation.id) -> bool:
+    '''
+    The function checks if the user already has a pair of such items to study that are in item_relation.
+    '''
+    item_relation: ItemRelation = await get_item_relation_db(item_relation_id)
+    card: Card = await Card.objects().where(
+        (Card.user.telegram_user_id == telegram_user_id) &
+        (Card.item_relation.item_1.is_in((item_relation.item_1, item_relation.item_2))) &
+        (Card.item_relation.item_2.is_in((item_relation.item_1, item_relation.item_2)))
+    )
```

Comment:
```suggestion
    card: Card = await Card.objects(Card.all_related()).where(
        (Card.user.telegram_user_id == telegram_user_id) &
        (Card.item_relation.item_1.id == item_relation_id)
    )
```
Мені здається такий запит буде працювати.

Reply:
І ще, здається що у card тип буде не Card,а багато Card. Навіть якщо ми впевнені, що там буде один запис.
Або додати до запиту .first()

Reply:
не розумію, як item's id можна порівнювати з item_relation's id, це різні сутності?

Reply:
Тип змінила.